### PR TITLE
Quiz rules and result page

### PR DESCRIPTION
**Pull request by tarunsankhla**

### CODE REVIEW

File: src/JS/JSResult.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                <label for="false">false</label>
+              </div>
+              
+              <div class="quiz-answer">
+                <input type="radio" id="undefined" name="js-question1" value="undefined">
+                <label for="undefined">undefined</label>
+              </div>
+              <div class="quiz-answer">
+                <input type="radio" id="Error" name="js-question1" value="Error">
+                <label for="Error">Error</label> -->
```
Comment (rahulyadav139) [inline src/JS/JSResult.html]: remove unnecessary comments from the file. Other than that great. Go ahead and merge the branch.

**Review by rahulyadav139 — APPROVED** [review]

File: src/CSS/CSS.css [hunk 2]
```diff
@@ -20,8 +22,10 @@
     padding: 2rem;
     line-height: 1rem;
     font-size: 2rem;
-
-    width: 80%;
+    width: 70%;
+    /* align-content: center; */
+    /* justify-content: center; */
```
Comment (PuneetP16) [inline src/CSS/CSS.css]: remove if not useful.

File: src/CSS/CSS.css [hunk 3]
```diff
@@ -36,8 +40,45 @@
...
+}
+
+.btn-outline:hover{
+    background-color: var(--warning-hover-color);
+}
+
 .code-snippet{
 display: inline-block;
     background-color: gray;
     padding: 1rem;
```
Comment (PuneetP16) [inline src/CSS/CSS.css]: Use variables instead of values. Will help in management.

**Review by PuneetP16 — APPROVED** [review]

**Review by saurabhchirde — APPROVED** [review]
The code is neat and clean, couldn't find any issue. 
Good to merge.

**Review by PuneetP16 — APPROVED** [review]
Good to merge.

**Comment by sushmita12345** [comment]
Looks good to merge.

**Review by sushmita12345 — APPROVED** [review]

File: src/CSS/CSSRules.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Round"
+    rel="stylesheet">
+    <!-- <link rel="stylesheet" href="https://tangent-ui.netlify.app/components/component.css" referrerpolicy="no-referrer"> -->
```
Comment (rahulyadav139) [inline src/CSS/CSSRules.html]: Files have many unnecessary comments. If these are not useful then remove these. Classnames are descriptive and simple. Great work.

**Review by rahulyadav139 — APPROVED** [review]

File: src/CSS/CSSRules.html (hunk 4 above)
Comment (tarunsankhla) [reply src/CSS/CSSRules.html]: okay removed

File: src/CSS/CSS.css (hunk 3 above)
Comment (tarunsankhla) [reply src/CSS/CSS.css]: okay done

File: src/CSS/CSS.css (hunk 2 above)
Comment (tarunsankhla) [reply src/CSS/CSS.css]: okay removed

**Pull request merged**